source/stylesheets/case-studies/easy-money/components/_image_showcase.scss

```diff
@@ -48,12 +48,30 @@
 
 .easy-ImageShowcase-main {
   z-index: 2;
+
+  @include media('<desktop') {
+    width: 50%;
```

Comment:
If the actual width you want is half the one sent from the server you should state this in picturefill as well, I mean in the `sizes` attribute in `_solution_create_voucher.slim`.